experimental/Schemes/CoherentSheaves.jl

```diff
@@ -158,46 +161,112 @@ identifications given by the glueings in the `default_covering`.
                                                        # and fⱼ the restrictions of the generators 
                                                        # of MD[V]. The aᵢⱼ are elements of 𝒪(U ∩ V)
                                                        # represented as a subset of V.
-      check::Bool=true
+      check::Bool=true,
+      default_cov::Covering=begin
+        patch_list = collect(keys(MD))
+        glueing_dict = IdDict{Tuple{AbsSpec, AbsSpec}, AbsGlueing}()
+        C = Covering(patch_list, glueing_dict)
+        inherit_glueings!(C, default_covering(X))
+        C
+      end
     )
     OOX = OO(X)
-    #OOX = StructureSheafOfRings(X)
+    # Make sure that all patches and glueings of the 
+    # given default_cov are compatible with the data in 
+    # the dictionaries.
+    all(x->haskey(MD, x), patches(default_cov)) || error("all patches in the default covering must have a prescribed module")
+    all(x->any(y->(x===y), patches(default_cov)), keys(MD)) || error("all prescribed modules must appear in the default covering")
+    all(x->haskey(MG, x), keys(glueings(default_cov))) || error("all glueings in the default covering must have a prescribed transition")
+    all(x->any(y->(x===y), keys(glueings(default_cov))), keys(MG)) || error("all prescribed transitions must correspond to glueings in the default covering")
+
+    ### Lookup and production pattern for sheaves of modules
+    #
+    # When asked to produce a module on an open affine U, the functions 
+    # below lead to the following behaviour.
+    #
+    # 1) Look up whether U is in the list of patches of default_cov.
+    #    If yes, return the value of the dictionary.
+    #
+    # 2) See whether U hangs below some V in the ancestry tree with V 
+    #    in the default_cov; if yes, restrict from V.
+    #
+    # 3) Otherwise, U is covered by patches {Vᵢ}, i ∈ I, from default_cov and 
+    #    contained in one affine chart W of X 
+    #      W ⊃ U ⊃ Vᵢ.
```

Comment:
This text sounds slightly unclear to me. Allow me to ask whether my understanding of it is correct:
It is the same W for all V_i, right?
This is the case where default_cov and W are completely unrelated (i.e. we do have a single affine chart W, but unfortunately it is not compatible with our default covering of X)?